version/version.go

```diff
@@ -1,9 +1,9 @@
 package version
 
 const (
-	// TMVersionDefault is the used as the fallback version of CometBFT
+	// CMTSemVer is the used as the fallback version of CometBFT
 	// when not using git describe. It is formatted with semantic versioning.
-	TMCoreSemVer = "0.39.0-dev"
+	CMTSemVer = "0.39.0-dev"
```

Comment:
These changes in the `version` package need a changelog entry, since they'll be breaking.